docs/creators/configure-forem.md

```diff
@@ -37,15 +37,15 @@ disappear.
 The following permissions are required to be able to view and/or edit the config
 page:
 
-#### Role: super_admin
+#### `Role: super_admin`
 
 When providing this role to a user they will be able to access the config page.
 However, this page will be a read only view for them. They will see the
 following:
 
 ![Super Admin Permissions Role Provided](https://dev-to-uploads.s3.amazonaws.com/i/xpc8g9x46vzgi49ohc0d.png)
 
-#### Role: single_resource_admin
+#### `Role: single_resource_admin`
```

Comment:
It seems like the underscores were somehow causing it to be italicised. Perhaps we could have tried escaping the underscores, but i'm okay with this for now too.